.codecov.yml

```diff
@@ -0,0 +1,23 @@
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
+
+  status:
+    project:
+      default:
+        enabled: yes
+        target: auto # must increase from parent commit or pull request base
+        if_no_uploads: error
+
+    patch:
+      default:
+        enabled: yes
+        if_no_uploads: error
+
+    changes: true
+
+
+comment:
+  layout: "header, diff, changes, tree, uncovered"
```

Comment:
The tree layout is a moderately new feature from CodeCov - you can see a screenshot here https://github.com/codecov/support/issues/171